afpacket/src/tokio_sockets.rs

```diff
@@ -20,11 +35,287 @@ impl AsyncBoundSocket {
         self.sock.get_mut().set_promiscuous(p)
     }
 
-    pub async fn send(&mut self, frame: &[u8]) -> io::Result<usize> {
-        self.sock.write(frame).await
+    pub fn split(self) -> (SendHalf, RecvHalf) {
```

Comment:
Documentation: Something about the purpose of split, what it is used for.  Reference to Tokio maybe.

Reply:
You don't have to go crazy here, but some documentation is needed for sure. Will help with maintenance in the future as well. :)